### PR TITLE
Feature/GitHub actions

### PR DESCRIPTION
**Pull request by misharigot**
- Added GitHub actions workflow for Python
  - Installs pip dependencies from `requirements.txt` 
  - Runs flake8 linter
  - Runs pytest tests

See https://github.com/wscs-2021/data-utils/actions for the **actions**.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```
Comment (steliosrammos) [inline .github/workflows/test.yml:23]: I believe pyyaml is a requirement too for the run.py script

**Review by steliosrammos — COMMENTED** [review]
Not familiar with Github actions at all so I can't review much :p I added a comment on the requirements, it is relevant for the README too.

File: .github/workflows/test.yml (hunk 1 above)
Comment (misharigot) [reply .github/workflows/test.yml:23]: Good point! This does not seem to be an issue for the actions runner though, perhaps its already included in the runner ubuntu image?

**Comment by misharigot** [comment]
> Not familiar with Github actions at all so I can't review much :p I added a comment on the requirements, it is relevant for the README too.

Yeah understandable. Just make sure to look at https://github.com/wscs-2021/data-utils/actions and try to understand what happens. 👍

**Pull request merged**